### PR TITLE
updated dev version and changelog

### PR DESCRIPTION
**Pull request by pauladkisson**
Post Release PR.

### CODE REVIEW

File: setup.py
Comment (h-mayorquin) [inline setup.py:25]: Maybe we should add Alessandra here?

**Review by h-mayorquin — APPROVED** [review]
Could you add a date?

File: CHANGELOG.md
Comment (h-mayorquin) [inline CHANGELOG.md:8]: here?

**Comment by pauladkisson** [comment]
@h-mayorquin, pls don't approve one of my PRs and then request changes in the comments. It's a pet-peeve of mine, and it can cause problems with auto-merge. For example, if all the tests pass and auto-merge is on, and you approve the PR, it will merge and close, and then I would have to open another PR to implement any feedback.

So pls, if you are requesting changes on a PR, don't approve the PR until I've actually implemented the changes.

**Comment by h-mayorquin** [comment]
We need some better language. If I approve, that mean that all the other things are suggestions if I feel very strong about something I would use the request button. 

How would you like me to communicate this?  Check here for guidelines in bigger projects:
https://scikit-learn.org/dev/developers/contributing.html#communication-guidelines

I do really like making suggestions that are not requestions or just discuss in PRs so just let me know do you prefer me to do that.

**Comment by pauladkisson** [comment]
> We need some better language. If I approve, that mean that all the other things are suggestions if I feel very strong about something I would use the request button.
> 
> How would you like me to communicate this? Check here for guidelines in bigger projects: https://scikit-learn.org/dev/developers/contributing.html#communication-guidelines
> 
> I do really like making suggestions that are not requestions or just discuss in PRs so just let me know do you prefer me to do that.

How about if you want to just make suggestions, you can select the "comment" option.  Then, you can expect me to either implement each suggestion or respond explaining why I don't want to, and once I've worked through all the comments I'll re-request your review. If your satisfied with my changes/responses, you can approve the PR at that point. If you want to continue the discussion we can do a back-and-forth until we reach agreement, and you can approve the PR at that point.

Would that work with your system?

**Comment by h-mayorquin** [comment]
Yeah, Paul, we are not that many, I can follow that policy when I interact with you. No comments after approval is a rule I can remember ; )

**Comment by pauladkisson** [comment]
Thanks for humoring me.

**Pull request merged**
